### PR TITLE
Clarification to assign-user-roles.md

### PR DESCRIPTION
**Pull request by sparkitect**
Testing indicates that I cannot add myself to a role but must have another admin do it for me (or sign in as another admin).
I've added a note to the document in case my test results are accurate...

### CODE REVIEW

File: Viva/insights/Setup/assign-user-roles.md [hunk 1]
```diff
@@ -41,6 +41,9 @@ Do the following to assign users to a Viva Insights role, including the **Insigh
 5. Enter the username, and then select the user from the list of suggestions. Optionally, add multiple users until you're done.
 6. Select **Save**.
 
+>[!Note]
+>You will not be able to assign yourself to a Role; another admin must do this for you.
```
Comment (JohanFreelancer9) [inline Viva/insights/Setup/assign-user-roles.md:45]:
```suggestion
> [!NOTE]
> You will not be able to assign yourself to a Role; another admin must do this for you.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @sparkitect, kindly commit the suggested changes. Many thanks.

**Comment by sparkitect** [comment]
I've found that this can actually be done (self-assignment) in Privileged Identity Management (Azure PIM). Should this also be noted, or kept to wizard-level knowledge.

**Review by JohanFreelancer9 — APPROVED** [review]
Copy review.

**Comment by scanum** [comment]
@madehmer
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request closed without merging**